api/errors/daemon.go

```diff
@@ -0,0 +1,21 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeNoSuchContainer is generated when we look for a container by
+	// name or ID and we can't find it.
+	ErrorCodeNoSuchContainer = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "NOSUCHCONTAINER",
+		Message:        "no such id: %s",
```

Comment:
shouldn't this be `no such id or name: %s`, or `no such container: %s`?

Also wondering if this could be grouped with other types since we have lookups for containers, images, volumes, and exec instances all with extremely similar cases.
Let me know if that's not the right usage of the package.